website/src/app/api.service.ts

```diff
@@ -178,7 +185,24 @@ export class ApiService {
       tap(data => console.log("closing issue", data)),
       catchError(this.handleError<Issue>("closeIssue", undefined)),
     );
-  }                                                                                                       
+  }      
+  
+  acknowledgeIssue(issue : Issue): Observable<Issue> {
+    return this.http.put<Issue>(`/api/v1/issues/${issue.id}/acknowledgeIssue`, undefined, {
+    }).pipe(
+      tap(data => console.log("acknowledging issue", data)),
+      catchError(this.handleError<Issue>("acknowledgeIssue", undefined)),
+    );
+  }
+
+  unacknowledgeIssue(issue: Issue): Observable<Issue> {
```

Comment:
Why does this function exist? Are we ever letting the users unacknowledge an issue?